release-notes/Web_apps/Web_apps_Feature_Release_10.4/Web_apps_Feature_Release_10.4.3.md

```diff
@@ -137,7 +137,13 @@ When, in the template editor, you pressed ESCAPE while the event editor window w
 
 <!-- MR 10.3.0 [CU12] - FR 10.4.3 -->
 
-A number of enum column issues have been fixed in the template editor.
+A number of enum column issues have been fixed in the Template Editor:
+
+- Feeding a query row to a component now includes both display values and values. This change helps avoid confusion when selecting a row visible across multiple tables.
+
+- Template conditions for string columns now rely on display values instead of column values, enhancing consistency and usability. This change improves interactions, especially when interacting with text inputs for discrete columns.
```

Comment:
```suggestion
- Template conditions for string columns now rely on the display values instead of the raw values, enhancing consistency and usability. This change improves interactions, especially when interacting with text inputs for discrete columns.
```